core/src/validator.rs

```diff
@@ -27,10 +27,12 @@ impl From<wasm::error::Error> for ValidationFail {
 
                 match call_error {
                     ExecutionLimitsExceeded(_) => Self::TooComplex,
-                    HostExecution(error) | Other(error) => Self::InternalError(error.to_string()),
+                    HostExecution(error) | Other(error) => {
+                        Self::InternalError(format!("{error:#}"))
+                    }
                 }
             }
-            _ => Self::InternalError(err.to_string()),
+            _ => Self::InternalError(format!("{err:#}")),
```

Comment:
How does this change the error message? Would it have proper newlines, or smth?

Reply:
It will print causes of error ([link](https://docs.wasmtime.dev/api/wasmtime/struct.Error.html#display-representations))

Reply:
Ah, that's just an anyhow error. Makes sense.